lib/runners/config.rb

```diff
@@ -35,6 +35,7 @@ def initialize(message, path_name:, raw_content:, attribute:)
 
     FILE_NAME = "sider.yml".freeze
     FILE_NAME_OLD = "sideci.yml".freeze
+    DEFAULT_METRICS_IGNORE_PATTERNS = ["**/vendor/**", "**/node_modules/**", "*.min.*", "*.map"].freeze
```

Comment:
[trivial] How about multi-line to reduce diff when new patterns will be added?

```suggestion
    DEFAULT_METRICS_IGNORE_PATTERNS = [
      "**/vendor/**",
      "**/node_modules/**",
      "*.min.*",
      "*.map",
    ].freeze
```